src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -6771,15 +6771,25 @@ End Class
             Dim intType As ITypeSymbol = comp.GetSpecialType(SpecialType.System_Int32)
 
             ' Illegal VB identifier and blank
-            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", ""))
-            Assert.Equal({"123", ""}, tuple2.TupleElementNames)
+            Dim tuple2 = comp.CreateTupleTypeSymbol(ImmutableArray.Create(intType, intType), ImmutableArray.Create("123", " "))
+            Assert.Equal({"123", " "}, GetTupleElementNames(tuple2))
```

Comment:
space is still legal, but blank is not allowed (since fields cannot have blank names)
I am adding separate tests for the "" case

---
In reply to: [86636919](https://github.com/dotnet/roslyn/pull/14958#discussion_r86636919) [](ancestors = 86636919)